File: Java/Day8.java
/*

Day 8: Program to print multiplication table of a number using for loop.

*/

// Program
import java.util.Scanner;

public class Day8{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number to print its table: ");
        int n = sc.nextInt();
        
        for(int i = 1; i <= 10; i++){
            System.out.println(String.format("%3d X %3d = %3d",n,i,n*i));
        }
    
    }
}

/*

Input/Output:

Enter a number to print its table: 12
 12 X   1 =  12
 12 X   2 =  24
 12 X   3 =  36
 12 X   4 =  48
 12 X   5 =  60
 12 X   6 =  72
 12 X   7 =  84
 12 X   8 =  96
 12 X   9 = 108
 12 X  10 = 120

*/

File: CPP/Day8.cpp
/* 

Day 8: Program to print multiplication table of a number using for loop

*/

//Program
#include <iostream>
using namespace std;

int main()
{
    int num;
    cout << "Enter a number : ";
    cin >> num;

    cout << "The Table of " << num << " is " << endl;
    for (int i = 1; i <= 10; i++)
    {
        cout << num << " x " << i << " = " << num * i << endl;
    }

    return 0;
}



/*

Input and Output

Enter a number : 7
The Table of 7 is
7 x 1 = 7
7 x 2 = 14
7 x 3 = 21
7 x 4 = 28
7 x 5 = 35
7 x 6 = 42
7 x 7 = 49
7 x 8 = 56
7 x 9 = 63
7 x 10 = 70

*/
